### ISSUE TITLE
Allow for more than Top5 users in replies

### ISSUE THREAD

**Issue by gedankenstuecke** [gedankenstuecke/twitter-analyser#13]

**Comment by minervapanda** [comment]
Hi, @gedankenstuecke I would like to work on this. I understand that we want to allow the visualization of replies from 5 to more -- should we allow the user to select the number of usernames he/she wants to visualize?

**Comment by gedankenstuecke** [comment]
@minervapanda yes, given the current setup this would mean having to process the data once for each number of users because so far the underlying data is only stored for the top5 users (see https://github.com/gedankenstuecke/twitter-analyser/blob/master/tweet_display/analyse_data.py#L101) 

Maybe we could store the Top 10 users instead and then just give a dropdown menu on the website so that people can filter down if they want to select fewer/more users (up to 10)?

**Comment by minervapanda** [comment]
Exactly what I had thought. Sounds good -- I will make a drop down for users to filter keeping the max. limit to 10.

**Comment by aashaka** [comment]
A checkbox with explicit ranking mentioned on its side to select users from Top 10 might be even more user-friendly.

**Comment by gedankenstuecke** [comment]
Yes, that sounds good too but the simpler version would be enough for a start 👍

**Comment by minervapanda** [comment]
Hi @aashaka Thanks for your suggestion. But checkbox doesnot look like a good design for the problem I am solving. I would give a drop-down with numbers 1-10 where selecting 1 means - selecting the top replied username's graph , 2 - top 2 usernames and so on. 
But, @gedankenstuecke can you please confirm me if the filter should be by number of top replies (1-10) or by username?

**Comment by gedankenstuecke** [comment]
@minervapanda filtering for the number of users to show  👍
